editor/store/effects.js

```diff
@@ -493,4 +497,13 @@ export default {
 		window.fetch( window._wpMetaBoxUrl, fetchOptions )
 			.then( () => store.dispatch( metaBoxUpdatesSuccess() ) );
 	},
+	EDIT_POST( action, { getState } ) {
+		const format = get( action, 'edits.format' );
+		if ( format ) {
```

Comment:
Minor: Early returns by negating condition help to avoid pyramid of indentation:

```js
const format = get( action, 'edits.format' );
if ( ! format ) {
	return;
}

// ...
```